### PR TITLE
BZ2093165: OpenStack compute and control plane machine requirements

### PR DESCRIPTION
**Pull request by xenolinux**
Version(s): 4.6+

Issue: https://bugzilla.redhat.com/show_bug.cgi?id=2093165

Link to docs preview: http://file.rdu.redhat.com/sdudhgao/stackshift-compute-machines/installing/installing_openstack/installing-openstack-installer-custom.html#installation-osp-compute-machines_installing-openstack-installer-custom

http://file.rdu.redhat.com/sdudhgao/stackshift-compute-machines/installing/installing_openstack/installing-openstack-installer-custom.html#installation-osp-control-machines_installing-openstack-installer-custom

Additional information:

QE: @eurijon

### CODE REVIEW

**Review by mandre — COMMENTED** [review]
LGTM from engineering.

**Comment by eurijon** [comment]
It looks good but added a minor comment, and also the PR description says this change is for OCP 4.7+, it's not for 4.6+?

**Comment by xenolinux** [comment]
> It looks good but added a minor comment, and also the PR description says this change is for OCP 4.7+, it's not for 4.6+?

Changed it to 4.6+

File: modules/installation-osp-control-compute-machines.adoc [hunk 1]
```diff
@@ -15,17 +15,29 @@ ifeval::["{context}" == "installing-openstack-installer-ovs-dpdk"]
 endif::[]
 
 [id="installation-osp-control-compute-machines_{context}"]
```
[the remark this answers is not in the document]
Comment (xenolinux) [reply modules/installation-osp-control-compute-machines.adoc]: Acked

**Comment by eurijon** [comment]
lgtm from QE

File: modules/installation-osp-control-compute-machines.adoc [hunk 2]
```diff
@@ -14,18 +14,30 @@ ifeval::["{context}" == "installing-openstack-installer-ovs-dpdk"]
...
 Each machine requires:
 
 * An instance from the {rh-openstack} quota
 * A port from the {rh-openstack} quota
 * A flavor with at least 16 GB memory, 4 vCPUs, and 100 GB storage space
 
+[id="installation-osp-compute-machines_{context}"]
+= Compute machines
+
+By default, the {product-title} installation process stands up three compute
```
Comment (sayjadha) [inline modules/installation-osp-control-compute-machines.adoc:32]: I'm not sure if this sounds like an instance of anthropomorphism.
Per the the IBM Style Guide: 
_"Anthropomorphism": Do not assign human characteristics to inanimate objects, which are incapable of human behaviors and emotions. As much as possible, focus technical information on users and their actions, not on a product and its actions._ 
Suggest rewording `stands up `-> `creates/generates` in all the instances, but I'll let you and another peer reviewer make the call.

**Review by sayjadha — COMMENTED** [review]
@xenolinux : I have one suggested edit although I'm not sure if it comes under anthropomorphism. Feel free to reject the suggestion if you think otherwise. Rest LGTM.

File: modules/installation-osp-control-compute-machines.adoc (hunk 2 above)
Comment (xenolinux) [reply modules/installation-osp-control-compute-machines.adoc]: Acked
